functions/zoom-downloader.py

```diff
@@ -374,3 +376,45 @@ def is_valid_mp4(filename):
     else:
         logger.debug("Successfully verified mp4 {}".format(url))
         return True
+
+
+def sort_files(files):
```

Comment:
I hate to do this after sitting on the PR for so long, but I'm not loving how the filtering and sorting is lumped together. Is it too complicating to break that up into two steps?